### PR TITLE
Nit picks

### PR DESCRIPTION
**Pull request by leobalter**

### CODE REVIEW

File: spec.html [hunk 1]
```diff
@@ -1682,7 +1682,7 @@ <h1>SetValueInBuffer ( _arrayBuffer_, _byteIndex_, _type_, _value_, _isTypedArra
           1. Assert: IsDetachedBuffer(_arrayBuffer_) is *false*.
           1. Assert: There are sufficient bytes in _arrayBuffer_ starting at _byteIndex_ to represent a value of _type_.
           1. Assert: _byteIndex_ is an integer value &ge; 0.
-          1. Assert: Type(_value_) is Number <ins>if _type_ is not `"BigInt64"` or `"BigUint64"`; otherwise, Type(_value_) is BigInt</ins>.
+          1. Assert: Type(_value_) is <ins>BigInt if _type_ is `"BigInt64"` or `"BigUint64"`; otherwise, Type(_value_) is Number</ins>.
```
Comment (leobalter) [inline spec.html]: I find this reading much easier when it's affirmative. you can discard this change if you don't like it.

File: spec.html [hunk 2]
```diff
@@ -1658,7 +1658,7 @@ <h1>IntegerIndexedElementSet ( _O_, _index_, _value_ )</h1>
           1. Assert: _O_ is an Object that has [[ViewedArrayBuffer]], [[ArrayLength]], [[ByteOffset]], and [[TypedArrayName]] internal slots.
           1. Let _arrayTypeName_ be the String value of _O_.[[TypedArrayName]].
           1. Let _elementType_ be the String value of the Element Type value in <emu-xref href="#table-49"></emu-xref> for _arrayTypeName_.
-          1. <ins>If _arrayTypeName_ is `"BigUint64Array"` or `"BigInt64Array"`, let _numValue_ be ? ToBigInt(_v_).</ins>
+          1. <ins>If _arrayTypeName_ is `"BigUint64Array"` or `"BigInt64Array"`, let _numValue_ be ? ToBigInt(_value_).</ins>
```
Comment (leobalter) [inline spec.html]: typo o/

File: spec.html (hunk 1 above)
Comment (littledan) [reply spec.html]: Nit: Technically, you'd want to not have the `<ins>` tag enclose the Number. Anyway, not very important.

File: spec.html (hunk 2 above)
Comment (littledan) [reply spec.html]: Good catch

**Pull request merged**